lesson_06/task_01/main.go

```diff
@@ -0,0 +1,22 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+)
+
+func headers(w http.ResponseWriter, req *http.Request) {
+	jdata := map[string]interface{}{"headers": req.Header, "user_agent": req.UserAgent(), "uri": req.RequestURI, "host": req.Host}
+	str, err := json.MarshalIndent(jdata, "", "    ")
+	if err != nil {
+		fmt.Printf("Error in json.Marshall: %v", err)
+	}
+	fmt.Fprintf(w, "%s\n", string(str))
+}
+
+func main() {
+	http.HandleFunc("/", headers)
```

Comment:
Please, add some logs about server, because when you run server, you didn't even know is server running or it stacked=)

Reply:
Done.